client/stripe-utils/__tests__/utils.test.js

```diff
@@ -0,0 +1,33 @@
+import { getFontSizeBase } from '../utils';
+
+describe( 'utils', () => {
+	describe( 'getFontSizeBase', () => {
+		const globalValues = global.wc_stripe_upe_params;
+
+		beforeEach( () => {
+			global.wc_stripe_upe_params = {
+				isOCEnabled: false,
+			};
+		} );
+
+		afterEach( () => {
+			global.wc_stripe_upe_params = globalValues;
+		} );
+
+		it( 'OC - should increase the provided font size by 2', () => {
```

Comment:
Nit: Can we use "Optimized Checkout" for clarity?
```suggestion
		it( 'Optimized Checkout - should increase the provided font size by 2', () => {
```

Reply:
And can we add a test case for decimals?

Reply:
Added in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4564/commits/56bb1b3b14e1dd6f8a6aeeb0a6441e397b4902ba